internal/gps/vcs_version.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package gps
+
+import (
+	"strings"
+
+	"github.com/Masterminds/vcs"
+	"github.com/pkg/errors"
+)
+
+// ProjectVersion returns the current project version for an absolute path.
+func ProjectVersion(path string) (Version, error) {
+	repo, err := vcs.NewRepo("", path)
```

Comment:
IIRC we have a gps-internal version of this call that uses our decorated types; as long as this is moving into gps, we should probably use that.